src/olympia/addons/tests/test_serializers.py

```diff
@@ -667,13 +668,166 @@ class TestAddonSerializerOutput(AddonSerializerOutputTestMixin, TestCase):
     serializer_class = AddonSerializer
     serializer_class_with_unlisted_data = AddonSerializerWithUnlistedData
 
+    def setUp(self):
+        super(TestAddonSerializerOutput, self).setUp()
+        self.action = 'detail'
```

Comment:
Isn't it 'retrieve'?

Reply:
It is, good catch. It doesn't matter anywhere at the moment but I'll fix it (the tests for the langpack version do use `retrieve` explicitly.)